packages/serverless-orchestration/README.md

```diff
@@ -0,0 +1,16 @@
+# @uma/serverless-orchestration
+
+This package contains scripts that enable bots to be run in a serverless fashion.
+## Installing the package
+
+```bash
+yarn add @uma/serverless-orchestration
+```
```

Comment:
Note: this package is somewhat complex to run locally (or at least when using it to run the bots). Would be interested to hear @chrismaree's opinion as to what we should put in this readme and what is out of scope.

Reply:
I think for now it should be out of scope. As a task for one of the following sprints, when looking into building out generalized bot infrastructure, we will refine this execution process and can improve the documentation then. 

For now, it's not really worth going into detail on how to run it for this reason I'd say.